### DISCUSSION TITLE
目前的chat和agent之类的是基于什么库开发的呢？

### DISCUSSION THREAD

**Discussion by MIngPAPA** [lobehub/lobehub#2080] [Development Question] [answered]
自从[Version 0.11.0](https://github.com/lobehub/lobe-chat/compare/v0.10.2...v0.11.0)， Langchain已经被移出了依赖。那么现在版本的lobe-chat的多轮对话、agent任务，都是如何实现的呢？

**Comment by arvinxx** [comment, marked as answer]
基于 ai sdk ，剩下的逻辑自行实现
